cmd/ocsp-updater/main.go

```diff
@@ -399,6 +399,9 @@ type looper struct {
 }
 
 func (l *looper) loop() {
+	if l.batchSize == 0 || l.tickDur == 0 {
+		return
```

Comment:
Please add an AUDIT log here, since we'll want to know if we mess up and leave it out.